### PR TITLE
gui-util: bugfix OdemisViewer Windows, read home folder properly

### PR DESCRIPTION
**Pull request by philipwinklerdelmic**

### CODE REVIEW

File: src/odemis/gui/util/__init__.py [hunk 1]
```diff
@@ -136,7 +136,15 @@ def get_home_folder():
     """ Return the home directory of the user running the Odemis GUI
     """
     # fall-back to HOME
-    folder = os.path.expanduser(u"~")
+    if sys.platform.startswith('linux'):
+        folder = os.path.expanduser(u"~")
+    elif sys.platform.startswith('win32'):
+        # Import functions here because wintypes in ctypes library cannot be opened in linux
```
Comment (pieleric) [inline src/odemis/gui/util/__init__.py]: More importantly, add a comment that expanduser(u) fails with non-ascii usernames (in Python2), and add a link to the bug report for Python.
Also, it'd be simpler to say that `winknownpath` cannot be imported if not on Windows.

File: src/odemis/gui/util/__init__.py [hunk 2]
```diff
@@ -136,7 +136,15 @@ def get_home_folder():
...
+        from odemis.gui.util.knownpaths import get_path, FOLDERID
+        folder = get_path(getattr(FOLDERID, "Profile"))
```
Comment (pieleric) [inline src/odemis/gui/util/__init__.py]: You probably can just write: `get_path(FOLDERID.Profile)`.
If so, replace the other calls too.

File: src/odemis/gui/util/knownpaths.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    def __init__(self, uuid_):
+        ctypes.Structure.__init__(self)
+        self.Data1, self.Data2, self.Data3, self.Data4[0], self.Data4[1], rest = uuid_.fields
+        for i in range(2, 8):
+            self.Data4[i] = rest>>(8 - i - 1)*8 & 0xff
+
+class FOLDERID:     # [2]
+    Pictures                = UUID('{33E28130-4E1E-4676-835A-98395C3BC3BB}')
+    Profile                 = UUID('{5E6C858F-0E22-4760-9AFE-EA3317B67173}')
```
Comment (pieleric) [inline src/odemis/gui/util/knownpaths.py]: For being slightly future-proof, could you add a couple of extra directories, like "Documents", "Program Files". Also, mention that the complete list is much longer.

**Comment by philipwinklerdelmic** [comment]
updated according to comments

**Pull request merged**